static/analytics.js

```diff
@@ -34,96 +33,53 @@ if (options.raven) {
     }).install();
 }
 
-var gaProxy;
-/* eslint-disable */
-if (options.googleAnalyticsEnabled) {
-    (function (i, s, o, g, r, a, m) {
-        i.GoogleAnalyticsObject = r;
-        i[r] = i[r] || function () {
-            (i[r].q = i[r].q || []).push(arguments);
-        };
-        i[r].l = 1 * new Date();
-        a = s.createElement(o);
-        m = s.getElementsByTagName(o)[0];
-        a.async = 1;
-        a.src = g;
-        m.parentNode.insertBefore(a, m);
-    })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
-    ga('create', options.googleAnalyticsAccount, 'auto');
-    ga('send', 'pageview');
-    gaProxy = function () {
-        window.ga.apply(window.ga, arguments);
-    };
-    // fullstory stuff:
-    window['_fs_debug'] = false;
-    window['_fs_host'] = 'www.fullstory.com';
-    window['_fs_org'] = '2F4NV';
-    window['_fs_namespace'] = 'FS';
-    (function (m, n, e, t, l, o, g, y) {
-        if (e in m && m.console && m.console.log) {
-            m.console.log('FullStory namespace conflict. Please set window["_fs_namespace"].');
-            return;
-        }
-        g = m[e] = function (a, b) {
-            g.q ? g.q.push([a, b]) : g._api(a, b);
-        };
-        g.q = [];
-        o = n.createElement(t);
-        o.async = 1;
-        o.src = 'https://' + _fs_host + '/s/fs.js';
-        y = n.getElementsByTagName(t)[0];
-        y.parentNode.insertBefore(o, y);
-        g.identify = function (i, v) {
-            g(l, {uid: i});
-            if (v) g(l, v);
-        };
-        g.setUserVars = function (v) {
-            g(l, v);
-        };
-        g.identifyAccount = function (i, v) {
-            o = 'account';
-            v = v || {};
-            v.acctId = i;
-            g(o, v);
-        };
-        g.clearUserCookie = function (c, d, i) {
-            if (!c || document.cookie.match('fs_uid=[`;`]*`[`;`]*`[`;`]*`')) {
-                d = n.domain;
-                while (1) {
-                    n.cookie = 'fs_uid=;domain=' + d +
-                        ';path=/;expires=' + new Date(0).toUTCString();
-                    i = d.indexOf('.');
-                    if (i < 0) break;
-                    d = d.slice(i + 1);
-                }
-            }
-        };
-    })(window, document, window['_fs_namespace'], 'script', 'user');
-} else {
-    gaProxy = function () {};
-}
-/* eslint-enable */
+function GAProxy() {
+    this.hasBeenEnabled = false;
+    this.isEnabled = false;
+
+    this.proxy = function () {};
 
-function initialise() {
-    if (options.embedded) return;
-    $(function () {
-        function create_script_element(id, url) {
-            var el = document.createElement('script');
-            el.type = 'text/javascript';
-            el.async = true;
-            el.id = id;
-            el.src = url;
-            var s = document.getElementsByTagName('script')[0];
-            s.parentNode.insertBefore(el, s);
+    this.initialise = function () {
```

Comment:
Nice! I was thinking this would be really tricky to do! You make it look easy! Thanks :)